src/elements/Table/Table.tsx

```diff
@@ -487,6 +487,13 @@ export class Table extends React.Component<IProps, State> {
                                                             />
                                                         </div>
                                                     )}
+                                                {item.input.type === 'number' && (
+                                                    <div>
+                                                        <input type="number"
```

Comment:
From my experience we should avoid `type="number"`, there's a lot of problems with it especially when it comes to comma vs dot in floats

Reply:
I agree, put it here for validation simplicity - should be changed/addressed after the MVP is over.

Reply:
Added a note `// TO-DO: Deprecate the use of input type number after POC`